lms/templates/pdf_viewer.html

```diff
@@ -27,7 +27,7 @@
 <html dir="ltr" mozdisallowselectionprint moznomarginboxes>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
See JIRA ticket for conversation about this. As @cptvitamin noted, this attribute is ignored on iOS. So I wasn't able to see any difference in behavior between staging and the sandbox.